src/plugins/intel_gpu/src/kernel_selector/kernels/fully_connected/fully_connected_kernel_bf_tiled.cpp

```diff
@@ -629,4 +745,161 @@ KernelsData FullyConnected_bf_tiled::GetKernelsData(const Params& params) const
+    }
+
+    bool succeed = UpdateWeightsParams(new_params,
+                                       wl,
+                                       kd.weightsReorderParams,
+                                       GetSupportedKey());
+    if (!succeed) {
+        return {};
+    }
+
+    int inputs_count = 1;
+    if (new_params.compressed) {
+        inputs_count++;
+        if (new_params.has_decompression_zp && !new_params.scalar_zp)
+            inputs_count++;
+    }
+
+    // Quantize kernel
+    const DispatchData dispatchData = SetDefault(new_params, autoTuneIndex, kernel_number);
+    {
+        auto& quan_kernel = kd.kernels[0];
+        DispatchData dyn_quan_dispatch = dispatchData;
+        dyn_quan_dispatch.gws = {std::max((fc_params.inputs[0].PhysicalSize() / quantize_grp_size), (size_t)1), 1, 1};
+        dyn_quan_dispatch.lws = {16, 1, 1};
+        quan_kernel.params.workGroups.global = dyn_quan_dispatch.gws;
+        quan_kernel.params.workGroups.local = dyn_quan_dispatch.lws;
+        quan_kernel.skip_execution = false;
+
+        // printf(" >> GetMultiKernelsData : Input0.Batch(%lu) Input0.Feature(%lu) Input0.Y(%lu) Input0.pysical(%lu)\n",
+        //         fc_params.inputs[0].Batch().v, fc_params.inputs[0].Feature().v, fc_params.inputs[0].Y().v, fc_params.inputs[0].PhysicalSize());
+        // printf("   -- Quantizing kernel : gws(%ld, %ld, %ld), lws(%ld, %ld, %ld)\n", dyn_quan_dispatch.gws[0], dyn_quan_dispatch.gws[1], dyn_quan_dispatch.gws[2],
+        //                                 dyn_quan_dispatch.lws[0], dyn_quan_dispatch.lws[1], dyn_quan_dispatch.lws[2]);
+
+        auto quan_entry_point = GetEntryPoint(kernelName, fc_params.layerID, params, kernel_number);
+        auto quan_cldnn_jit = GetJitConstants(new_params, dyn_quan_dispatch);
+        quan_cldnn_jit.AddConstant(MakeJitConstant("FC_KERNEL_DYNAMIC_QUANTIZE", 1));
+        auto quan_jit = CreateJit(kernelName, quan_cldnn_jit, quan_entry_point);
+
+
+        FillCLKernelData(quan_kernel,
+                        dyn_quan_dispatch,
+                        params.engineInfo,
+                        kernelName,
+                        quan_jit,
+                        quan_entry_point,
+                        exeMode,  // No exec mode
+                        false,
+                        false,
+                        1, // Only INPUT_0 is used for quantizing
+                        0, // No fused ops
+                        0, // No output
+                        fc_params.is_shape_agnostic);
+
+        quan_kernel.params.arguments.clear();  // Clear original output argument
+        quan_kernel.params.arguments.push_back({ArgumentDescriptor::Types::INPUT, 0});
+        quan_kernel.params.arguments.push_back({ArgumentDescriptor::Types::INTERNAL_BUFFER, 0});
+        quan_kernel.params.arguments.push_back({ArgumentDescriptor::Types::INTERNAL_BUFFER, 1});
+        kd.internalBufferSizes.push_back(fc_params.inputs[0].PhysicalSize() * 2);
```

Comment:
I think `fc_params.inputs[0].PhysicalSize()` instead of `fc_params.inputs[0].PhysicalSize() * 2` because it is int8

Reply:
Fixed